modules/security-lti/src/main/java/org/opencastproject/security/lti/LtiLaunchAuthenticationHandler.java

```diff
@@ -237,9 +239,8 @@ protected void activate(ComponentContext cc) {
       usernameBlacklist.add(username);
     }
 
-    createJpaUserReference = BooleanUtils.toBooleanDefaultIfNull(
-      BooleanUtils.toBooleanObject(StringUtils.trimToNull((String) properties.get(CREATE_JPA_USER_REFERENCE_KEY))),
-      true);
+    ltiRolesForUserCreation = extractLtiRolesForUserCreation(
+            (String) properties.get(LTI_ROLES_TO_CREATE_JPA_USER_REFERENCES_FROM));
```

Comment:
Actually set the default you claim to set:

```suggestion
    ltiRolesForUserCreation = extractLtiRolesForUserCreation(
            Objects.toString(properties.get(LTI_ROLES_TO_CREATE_JPA_USER_REFERENCES_FROM), "*");
```

Reply:
The default (*) is set programmatically now.